### PR TITLE
fix: squads mvp mobile page responsiveness

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- Notification modal to change the title.
- Name and handle being centered.
- Avoid using ClickableText on links (we might need a new component).

### Describe what this PR does
- Short and concise, bullet points can help
- Screenshots if applicable can also help

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1613 #done

### CODE REVIEW

File: packages/shared/src/components/modals/squads/SquadNotificationsModal.tsx [hunk 1]
```diff
@@ -137,16 +136,14 @@ export function SquadNotificationsModal({
             Notify me about new members
           </Switch>
         )}
-        <p className="flex flex-row typo-callout text-theme-label-tertiary">
+        <p className="typo-callout text-theme-label-tertiary">
           For more options, go to your
-          <ClickableText
```
Comment (sshanzel) [inline packages/shared/src/components/modals/squads/SquadNotificationsModal.tsx:142]: Using this doesn't break the words no matter what CSS property I applied due to the Button's base classes.

Comment (rebelchris) [reply packages/shared/src/components/modals/squads/SquadNotificationsModal.tsx:142]: Yeah I also had some issues with this component, for now your solution is good.
Maybe we can add a backlog task for a new component?

**Review by rebelchris — APPROVED** [review]
Looks good.

File: packages/shared/src/components/modals/squads/SquadNotificationsModal.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/components/modals/squads/SquadNotificationsModal.tsx:142]: Logged! 🚢

**Pull request merged**